test/CMakeLists.txt

```diff
@@ -157,7 +176,7 @@ addLinkAndDiscoverTest(PrefixCompressorTest)
 
 addLinkAndDiscoverTest(MilestoneIdTest absl::strings)
 
-addLinkAndDiscoverTest(VocabularyOnDiskTest index)
+addLinkAndDiscoverTestSerial(VocabularyOnDiskTest index)
```

Comment:
This can also cause a `BusError`. It seems that all the test cases use the same name for the file on disk with `VocabularyOnDisk`. Is `VocabularyOnDisk` maybe also doing memory mapping?

Reply:
You are right, it is indeed. Here we could possibly fix it by giving the indices distinct basenames,
So comment "The `VocabularyOnDisk` uses mmap which fails when the tests are run in parallel. This should be fixed by using distinct filenames for each test case".